cryptocurrency-subdomain/src/main/kotlin/com/ajax/cryptocurrency/application/mapper/CryptocurrencyMapper.kt

```diff
@@ -0,0 +1,25 @@
+package com.ajax.cryptocurrency.application.mapper
+
+import com.ajax.cryptocurrency.domain.CryptocurrencyDomain
+import com.ajax.cryptocurrency.infrastructure.mongo.entity.CryptocurrencyEntity
```

Comment:
entity is an infra layer which should not be used in application layer

Reply:
moved to infra layer